src/System.CommandLine/ArgumentArity.cs

```diff
@@ -72,14 +85,29 @@ public ArgumentArity(int minimumNumberOfValues, int maximumNumberOfValues)
             return null;
         }
 
+        /// <summary>
+        /// Must not have any arguments.
```

Comment:
Maybe something like: "An arity that does not allow any values."

Reply:
Done